back/engines/commercial/report_builder/app/models/report_builder/report.rb

```diff
@@ -29,6 +32,8 @@ class Report < ::ApplicationRecord
     )
     accepts_nested_attributes_for :layout
 
-    validates :name, presence: true, uniqueness: true
+    scope :with_platform_context, -> { where(participation_context_type: nil) }
```

Comment:
@luucvanderzee It's how I call the reports w/o any participation context. Maybe you'll need this name too. I'm also open to changing it if you think it's unclear.

Reply:
Makes sense, I don't see why we would need a name for these type of reports

Reply:
I wasn't super clear. With my comment, I meant that I call reports that do not belong to phases (or any other participation context) by the name of `Reports with (entire) platform participation context`.